src/pages/create-event.js

```diff
@@ -1,115 +1,370 @@
+    const [ optArr, setOptArr ] = useState(["", "", "", ""]);
+    const [ date, setDate ] = useState(new Date());
+    const [startTime, setValue] = React.useState(new Date());
+    const [ endTime, setEndTime ] = React.useState(new Date());
+    const [ timeZone, setTimeZone ] = useState("CST");
+    const [ vError, setVError ] = useState({});
+    const [ inputFields, setInputFields ] = useState([
+        { name: '', email: '' },
+    ])
+   
+    const handleChange = (newValue) => {
+      setValue(newValue);
+    };
+
+    const handleChangeInput = (index, event) =>{
+        const values = [...inputFields];
+        values[index][event.target.name] = event.target.value;
+        setInputFields(values);
+    }
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
     }
+
+    const handleAddFields = () => {
+        setInputFields([...inputFields, { name: '', email: '' }])
+    }
+
+    const handleRemoveFields = (index) => {
+        const values = [...inputFields];
+        values.splice(index, 1);
+        setInputFields(values);
+    }
+
+    // only changes timezone label
+    const changeTimeZone = e => {
+        setTimeZone(e.target.value);
+        console.log(e.target.value);
+    };
     
+      
+    const validate = () => {
+        const errors = {};
+        
+        if (title.trim() === "") {
+            errors.title = "Title needed!";
+        }
+
+        setVError(errors);
+        return errors;
+    }
+
+    const handleSubmission = e => {
+        e.preventDefault();
+
+        const err = validate();
+        console.log("Err is", err);
+        if (Object.keys(err).length !== 0) {
+            let errMsg = "";
+            for (let key in err) {
+                errMsg = errMsg.concat(err[key]).concat("\n");
+            }
+            alert(errMsg);
+            console.log("error");
+            return;
+        }
+        console.log("valid");
+        
+        console.log("InputFields", inputFields);
+        const temp =  JSON.stringify(date);
+        const temp2 =  JSON.stringify(startTime);
+        const temp3 =  JSON.stringify(startTime);
+        const newEvent = {
+
+            title: title,
+            start: 
+                [
+                    parseInt(temp.slice(0,4)),
+                    parseInt(temp.slice(5,7)),
+                    parseInt(temp.slice(8,10)),
+        
+                    parseInt(temp2.slice(0,2)),
+                    parseInt(temp2.slice(3))
+                ],
+            end : 
+                [
+                    parseInt(temp.slice(0,4)),
+                    parseInt(temp.slice(5,7)),
+                    parseInt(temp.slice(8,10)),
+            
+                    parseInt(temp3.slice(0,2)),
+                    parseInt(temp3.slice(3))
+                ],
+            timeZone: timeZone,
+            attendants: inputFields, 
+            allDay: false
+        };    
+      
+        console.log(timeZone);
+        writeEvent(newEvent);
```

Comment:
So the issue here is that you're not calling writeEvent with the parameters it is expecting. You can look at the comments in that function utils/events.js

Hopefully that clears up the database write issue. Let me know if you have any questions about that.